resources/teamcity/web/keyman-web-release.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # Copyright (C) 2025 SIL International. All rights reserved.
```

Comment:
We should be usin the [standard header](https://github.com/keymanapp/keyman/wiki/Keyman-Code-Style-Guide#file-header) in all our scripts